tools/eslint-plugin-cdk/test/rules/fixtures/invalid-cfn-imports/cfn-constructor.error.txt

```diff
@@ -0,0 +1 @@
+To allow rewriting imports when generating v2 experimental packages, `CfnDeliveryStream` must be imported by name and separate from non-L1 imports, since it is being imported from an experimental package: @aws-cdk/aws-kinesisfirehose
```

Comment:
Why only one error message? It should be possible to write tests that produce multiple error messages and verify that.

And it should be order insensitive.

Reply:
I implemented this to be one line per error message in the `error.txt` file, let me know if you have thoughts to make that format better! csv, maybe?

Reply:
How about one error message per line?